### DISCUSSION TITLE
routingMode and startingView

### DISCUSSION THREAD

**Discussion by lephtHanded** [Lissy93/dashy#542] [Help and Support] [answered]
heya!
latest update (or my config) seems borked with the latest docker pull.
the `routingMode` and `startingView` options don't seem to have an effect.

i have mine set to `hash` and `minimal` respectively and the page loads to the default view.

links to `../minimal` or `../#/minimal` that I got working from [this thread](https://github.com/Lissy93/dashy/discussions/148#discussioncomment-1660414_) just open the default view now.

**Comment by lissy93** [comment]
Ah yes, seems that the config isn't finished fetching when the [`router`](https://github.com/Lissy93/dashy/blob/master/src/router.js) is initialised. Sorry about that, lemme convert this to an issue... #544

**Comment by azerioxal** [comment, marked as answer]
Sorry about that. I'm getting started on fixing that now.

**Reply by lephtHanded** [reply]
It works!!!  
thank you all!
